your-first-cluster.md

```diff
@@ -0,0 +1,512 @@
+
+```bash
+export CLUSTER_TOPOLOGY=true
+clusterctl init --infrastructure docker:v1.2.2
+```
+
+Each of the provider controllers is installed as a pod on the Kind cluster. Once they're up and running you have your very own management Cluster!
+
+To verify:
+```bash
+kubectl get pods -A
+```
+Output:
+```bash
+capd-system                         capd-controller-manager-6f9c9d56b-h722b                          1/1     Running   0          84s
+capi-kubeadm-bootstrap-system       capi-kubeadm-bootstrap-controller-manager-75dcdf5f8-qdrl5        1/1     Running   0          85s
+capi-kubeadm-control-plane-system   capi-kubeadm-control-plane-controller-manager-8685fc89d4-fdp5s   1/1     Running   0          84s
+capi-system                         capi-controller-manager-6c7689f5b9-2rn2v                         1/1     Running   0          85s
+cert-manager                        cert-manager-86b6c9bf4b-f7hc6                                    1/1     Running   0          101s
+cert-manager                        cert-manager-cainjector-7d4868cf69-k8jx5                         1/1     Running   0          101s
+cert-manager                        cert-manager-webhook-6cb6fbfc58-qzpvg                            1/1     Running   0          101s
+kube-system                         coredns-6d4b75cb6d-8trn8                                         1/1     Running   0          5m27s
+kube-system                         coredns-6d4b75cb6d-f76kw                                         1/1     Running   0          5m27s
+kube-system                         etcd-kind-control-plane                                          1/1     Running   0          5m42s
+kube-system                         kindnet-l6qp2                                                    1/1     Running   0          5m27s
+kube-system                         kube-apiserver-kind-control-plane                                1/1     Running   0          5m39s
+kube-system                         kube-controller-manager-kind-control-plane                       1/1     Running   0          5m39s
+kube-system                         kube-proxy-2zgh8                                                 1/1     Running   0          5m27s
+kube-system                         kube-scheduler-kind-control-plane                                1/1     Running   0          5m40s
+local-path-storage                  local-path-provisioner-9cd9bd544-sglzs                           1/1     Running   0          5m27s
+
+```
+
+## Your first workload cluster
+
+So what can you do with a functioning management cluster? Create more Clusters! The first step in doing this is to create a `ClusterClass` - this is a template that defines the shape of your cluster.
+
+```bash
+kubectl apply -f yamls/clusterclasses/clusterclass-quick-start.yaml
```

Comment:
Technically we could also use the generate cmd of clusterctl with the --config thing to have it all offline. But your flow here is better I think

(just resolve if you agree)